Azure/connectors/trigger_azure_eventhub.py

```diff
@@ -70,14 +70,6 @@ def __init__(self, *args: Any, **kwargs: Optional[Any]) -> None:
         self._stop_event = Event()
         self._consumption_max_wait_time = int(os.environ.get("CONSUMER_MAX_WAIT_TIME", "600"), 10)
 
-    async def stop(self, *args: Any, **kwargs: Optional[Any]) -> None:
-        self.log(message="Stopping Azure EventHub connector", level="info")
-        # Exit signal received, asking the processor to stop
-        self._stop_event.set()
-
-        # Close the azure EventHub client
-        await self.client.close()
```

Comment:
Arf. We will not properly close the client. It may cause some issues in the consumption of the event hub.

Reply:
I don't believe we're actually calling `stop()` anywhere. I overrode the method in order to use callbacks for SIGINT/SIGTERM. But since we use an existing loop from AsyncConnector, we don't have to set those up.